e107_handlers/date_handler.php

```diff
@@ -937,8 +938,7 @@ public function dateFormats($datestamp = null)
 						
 						"%Y-%m-%d",
 						"%d-%m-%Y",
-						"%m/%d/%Y",
-						"%d.%m.%Y"
+						"%m/%d/%Y"
```

Comment:
@Jimmi08 Please explain the removal. Thanks.

Reply:
It wasn't intention...  I didn't notice this. Please, either change it or close and I do it again. Thanks for noticing.